calour/experiment.py

```diff
@@ -15,6 +15,8 @@
 import scipy
 import biom
 
+import calour as ca
+
 
```

Comment:
this made calour work (so i can use the read and test the functions i'm writing...)
maybe you/i can merge the PRs and resolve the conflicts?
what can we do?